c10/core/SymFloat.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <c10/core/SymFloatNodeImpl.h>
+#include <c10/macros/Macros.h>
+#include <c10/util/Exception.h>
+#include <c10/util/intrusive_ptr.h>
+
+#include <limits>
+#include <memory>
+
+namespace c10 {
+
+// NB: this is actually double precision; we're using the Python naming here
+class C10_API SymFloat {
```

Comment:
Why did you choose to use python naming here? It's ok with the comment but generally I would likely forget that this was meant to represent double if I saw this at an arbitrary line in the C++ code.

Reply:
It's for consistency with SymInt, which is also Python naming (int is typically 32-bit) and the JIT type system which calls it float.